core/src/components/item/usage/vue.md

```diff
@@ -410,3 +410,38 @@ export default defineComponent({
 });
 </script>
 ```
+
+### Item Counter
+
+```html
+<template>
+  <ion-item :counter="true" :counter-formatter="counterFormatter">
```

Comment:
Should these two examples be reversed? The second one says "item counter formatter" but does not use the formatter.

Reply:
Looks like this also happens elsewhere